lib/build_cache.py

```diff
@@ -269,6 +271,10 @@ def __init__(self, image_root, path):
       self.dont_restore = False
       self.hardlink_to = None
       self.large_name = None
+      self.xattrs = dict()
+      if ch.xattrs:
```

Comment:
I wonder if this variable should be a little more descriptive? Is this a boolean saying whether or not to save xattrs?